docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -118,6 +118,14 @@ Each action type exposes different properties. For example, an email action allo
 [role="screenshot"]
 image::images/rule-flyout-action-details.png[UI for defining an email action]
 
+You can attach more than one action. Clicking the *Add action* button will prompt you to select another rule type and repeat the above steps again.
+
+[NOTE]
+==============================================
+Actions are not required on rules. You can run a rule without actions to
+understand its behavior, then <<action-settings,configure actions>> later.
```

Comment:
This link goes to the  `actions` kml configuration page, which I don't think was the intent.  I'm not exactly where we should send them, I think to the top of this section?  So, no link really needed at all?

Reply:
IMO rather than having this at the end of the section as a note, we could integrate it into the first sentence in this section. For example:

> Actions are optional when you create a rule. However, to receive notifications when a rule meets the defined conditions, you must add one or more actions...

If you want me to edit this PR with that change, just let me know!

Reply:
@lcawl yes please on help reformatting 🙏🏼